src/storage/src/hummock/compactor/compactor_runner.rs

```diff
@@ -507,6 +511,13 @@ pub async fn compact(
             cost_time,
             compact_task_to_string(&compact_task)
         );
+        // TODO: remove this method after we have running risingwave cluster with fast compact algorithm stably for a long time.
+        if let Err(e) = check_compaction_result(&compact_task, context.clone()).await {
```

Comment:
Add configuration to control whether the `check` is turned on to avoid affecting the normal process.